This fixes an invalid `Component::getAbsolutePathString` method call in
`OpenSim::Output<T>::Channel::getPathName`, which uses the `Component` API
before it has been defined.

When compiling with MSVC, this can cause an error when evaluating C++20
concepts around the templated class - even if the offending method is never
actually called.

---
diff --git a/OpenSim/Common/ComponentOutput.h b/OpenSim/Common/ComponentOutput.h
index 62ea6cf53..18e32eaff 100644
--- a/OpenSim/Common/ComponentOutput.h
+++ b/OpenSim/Common/ComponentOutput.h
@@ -355,7 +355,7 @@ public:
         return getOutput().getName() + ":" + _channelName;
     }
     std::string getPathName() const override {
-        return getOutput().getOwner().getAbsolutePathString() + "|" + getName();
+        return getOutput().getPathName();
     }
 private:
     mutable T _result;
